src/cmd/builtin/filemap.c

```diff
@@ -375,9 +375,9 @@ static struct optparse_option map_opts[] = {
       .usage = "Change to DIR before mapping", },
     { .name = "verbose", .key = 'v', .has_arg = 2, .arginfo = "[LEVEL]",
       .usage = "Increase output detail.", },
-    { .name = "chunksize", .has_arg = 1, .arginfo = "N",
+    { .name = "chunksize", .has_arg = 1, .arginfo = "N[kMG]",
```

Comment:
intentionally only document kMG here?  It does make sense, perhaps don't want to intentionalllly suggest exabyte chunksize.